### PR TITLE
Henry's information added

### PR DESCRIPTION
**Pull request by HSecaira**
Dear Dr. Zhu, 

I added my cv, photo, and information to the people.md file. 

Let me know if I must change something. Thanks!

### CODE REVIEW

**Comment by qiyunzhu** [comment]
@HSecaira Very good! Welcome on board! Just a trivial note: Your picture is in portrait direction with black edges on top and bottom, which may stretch the layout in the "People" page. Would you mind trimming the edges?

**Comment by HSecaira** [comment]
@qiyunzhu  I have cropped my photo. Let me know if it works. Thanks!

File: people.md [hunk 1]
```diff
@@ -34,6 +34,17 @@ postdoc:
       [<i class='fab fa-github-square fa-2x'></i>](https://github.com/yujulian0168)&nbsp;&nbsp;
       [<i class='fab fa-linkedin fa-2x'></i>](https://www.linkedin.com/in/julian-yu-38836a141/)&nbsp;&nbsp;
       [<i class='ai ai-cv-square ai-2x'></i>](assets/files/YU,J-CV2020.pdf)
+
+phd:
+  - title: Henry Secaira Morocho
+    image_path: assets/images/people/HenryPhoto.jpeg
```
Comment (qiyunzhu) [inline people.md:40]: @HSecaira Don't forget to modify the image file name too since your cropped image has a different name.

**Review by qiyunzhu — CHANGES_REQUESTED** [review]

**Comment by HSecaira** [comment]
@qiyunzhu sorry about that. The image file name has been modified. Let me know if there is something else I should change. Thanks!

**Pull request merged**